.github/workflows/devtools.yml

```diff
@@ -93,11 +93,15 @@ jobs:
       with:
         node-version: 14.x
 
-    - name: Download artifact
-      uses: actions/download-artifact@v2
+    - name: Cache build artifacts
```

Comment:
What do you think of renaming these two? `Load cached build artifacts` or something? Less important for the logs, but kind of confusing when reading this file